js/components/main.js

```diff
@@ -224,11 +224,11 @@ class Main extends ImmutableComponent {
       <div className='top'>
         <div className='navigator-wrapper'>
           <div className='backforward'>
-            <span
+            <span title='Go back'
```

Comment:
@bbondy updated with data-l10n-ids 

let me know if there is some kind of system in the locale file that I've missed, I just put them near related items